components/waves/memes/submission/hooks/useResubmissionDelete.ts

```
@@ -0,0 +1,127 @@
+        message = error.message;
+      } else if (typeof error === "string") {
+        message = error;
+      }
+      setDeleteOriginalError(message);
+      setToast({
+        message,
+        type: "error",
+      });
+    } finally {
+      if (didClose) {
+        return;
```

Biome (2.4.11)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)